jrnl/DayOneJournal.py

```diff
@@ -210,7 +210,7 @@ def parse_editable_str(self, edited):
         entries_from_editor = self._parse(edited)
 
         for entry in entries_from_editor:
-            entry = self._get_and_remove_uuid_from_entry(entry)
+            self._get_and_remove_uuid_from_entry(entry)
```

Comment:
I know this is basically a no-op, but can we avoid anything other than type hint changes in this PR (to keep it separate from everything else)?

Reply:
Sure. Just for reference, there were two other changes not related to type hints:
- https://github.com/jrnl-org/jrnl/pull/1614/files#diff-db81d051c76b672dfb78a301ae64d1b68a092ea692df1b20e28d2e44de2a2f7fL48-R56
- https://github.com/jrnl-org/jrnl/pull/1614/files#diff-d01e148158bf7a61dcec573061bc9ade91eeb795a21c47a0edbf8e6561f4fb78L118-R118